pkg/controller/plan/validation.go

```diff
@@ -491,6 +514,9 @@ func (r *Reconciler) validateVM(plan *api.Plan) error {
 	if len(multiplePodNetworkMappings.Items) > 0 {
 		plan.Status.SetCondition(multiplePodNetworkMappings)
 	}
+	if len(staticStorage.Items) > 0 {
+		plan.Status.SetCondition(staticStorage)
+	}
```

Comment:
this means that when a user defines a plan that is going to use a volume-populator and storage map that includes an invalid target namespace, the operation will succeed and only later set a condition on the plan - why not to block this with a validation webhook if we're talking about an invalid input that we can check before persisting the plan?